executor/builder.go

```diff
@@ -1354,12 +1354,16 @@ func (b *executorBuilder) buildMemTable(v *plannercore.PhysicalMemTable) Executo
 				},
 			}
 		case strings.ToLower(infoschema.TableSlowQuery), strings.ToLower(infoschema.ClusterTableSlowLog):
+			retriever := &SlowQueryRetriever{
+				table:      v.Table,
+				outputCols: v.Columns,
+			}
+			if v.Extractor != nil {
```

Comment:
Is it necessary? seems the `v.Extractor` always be assigned a value (https://github.com/pingcap/tidb/pull/14840/files#diff-b9e75a99170ea35a67074d5d3d3305adR2800).

Reply:
No, `ClusterTableSlowLog ` will not now.
Next PR will make `ClusterTableSlowLog` support this, then we can remove it.